src/main/webapp/dataverse_header.xhtml

```diff
@@ -23,11 +23,12 @@
                         <span class="icon-bar"></span>
                     </button>                       
                     <a href="/">
-                        <ui:fragment rendered="#{empty settingsWrapper.get(':LogoCustomizationFile')}">
+                        <ui:fragment rendered="#{empty settingsServiceBean.get(':LogoCustomizationFile')}">
```

Comment:
I wouldn't switch usage of `settingsWrapper` to `settingsServiceBean` here and in all xhtml cases.
SettingsWrapper is view specific and SettingsServiceBean is not